--- scripts/build.py	2014-12-04 14:01:56.627961566 +0000
+++ scripts/build_fixtar.py	2014-12-04 14:02:24.671238061 +0000
@@ -1055,7 +1055,7 @@
     shell('cp ../../../include/wkhtmltox/*.h ../wkhtmltox-%s/include/wkhtmltox' % version)
     shell('cp ../../../include/wkhtmltox/dll*.inc ../wkhtmltox-%s/include/wkhtmltox' % version)
 
-    os.chdir(basedir)
+    os.chdir(os.path.join(basedir, config))
     shell('tar -c -v -f ../wkhtmltox-%s_local-%s.tar wkhtmltox-%s/' % (version, platform.node(), version))
     shell('xz --compress --force --verbose -9 ../wkhtmltox-%s_local-%s.tar' % (version, platform.node()))
 
